serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-grpc-services.adoc

```diff
@@ -14,20 +13,30 @@ As a developer, you can invoke a gRPC service using a Serverless Workflow applic
 
 For information about gRPC concepts, see link:{grpc_core_url}[gRPC].
 
-In order to illustrate how gRPC integration works, this link:{grpc_example_url}/serverless-workflow-greeting-client-rpc-quarkus[Serverless Workflow application] is used. This example greets a user in their preferred language by invoking a gRPC service. 
+In order to illustrate how gRPC integration works, this link:{grpc_example_url}/serverless-workflow-greeting-client-rpc-quarkus[Serverless Workflow application] is used. This example greets a user in their preferred language by invoking a gRPC service.
 
 Relevant files of this Workflow Application are:
 
-* link:{grpc_example_url}/serverless-workflow-greeting-server-rpc-quarkus/src/main/proto/greeting.proto[gRPC proto file]: 
+* link:{grpc_example_url}/serverless-workflow-greeting-server-rpc-quarkus/src/main/proto/greeting.proto[gRPC proto file]:
 
-This file defines a greeter service, which consists of the `sayHello` method. The `sayHello` method accepts two parameters, including name of the user and an optional language to use in the greeting message (English by default). The `sayHello` method returns the greeting message in the appropriate language.
+This file defines a greeter service, which consists of sayHello methods, each covering a different gRPC scenario:
+
+1. The `sayHello` method accepts two parameters, including name of the user and an optional language to use in the greeting message (English by default). The `sayHello` method returns the greeting message in the appropriate language.
+2. The `SayHelloAllLanguages` accepts just the name parameter and streams greetings in all languages supported by the service. This represents the server-side streaming scenario.
+3. The `SayHelloMultipleLanguagesAtOnce` accepts a stream of name-language pairs as parameters and returns corresponding greetings as one block of text after the streaming is finished. This is the client-side streaming scenario.
+4. The `SayHelloMultipleLanguages` accepts a stream of name-language pairs and streams back greetings with a respective name and in a respective language.
+5. The `SayHelloMultipleLanguagesError` is the same as before but is used just for testing purposes. It is not relevant for this guide.
```

Comment:
If it is not relavant for this guide, we can skip mentioning it. 
We are not describing the proto, just mentioning those things in the proto which are relevant to the guide

Reply:
Fixed.